src/LayoutManager/README.md

```diff
@@ -2,6 +2,39 @@
 
 Layout manager exposes a main public method that performs group layout given a target group and target objects.
 
+## Expectations for non interactive groups
+
+The layout manager will calculate the size and position of the group when is created or updated
+The FitContent strategy is the classic fabric 5.x group and will grow or shrink when objects are added or removed.
+In general the expectation is that creating or modifying an object will create a group with a postion and once that group is added to canvas, the position of the objects will be the same visually that those object would have had on the canvas if they weren't on the group.
+This is a legacy behaviour that carries over this new code.
+The behaviour is not important if you are building a group from scratch and you are adding it in a specific position on the canvas, in that case the original position is meaningless and you are overriding it by specifying your postion.
+
+Since inside the group all objects are organized around the center is anyway easier if you can think of your objects position and alignment with scene coordinates and know that they will be maintained in the same position when grouped.
+
+### FitContent strategy
+
+When a group is created, FitContent strategy will consider all the objects in the group, find their bounding box, find the center of the bounding box, and assign those to the group.
+
+When a new object is added or removed, a new bounding box is calculated, a new center is found, all the remaining objects are moved by an offset so that are re-centered around the new center
+
+### Fixed layout strategy
+
+With the fixed layout a Developer can specify either width or height and have the other calculated.
+If the developer specifies both dimensions of the box, the bounding box is still calculated to obtain the center.
+Once the group is initialized with a fixed layout, adding new objects is not changing the center anymore.
+The layout is fixed, is cropped by the bounding box itself.
+When larger or smaller than the actual boundingbox the group is centered on the initial center of the initial set of objects and the fixed bounding box extends from the center.
```

Comment:
wrong
it used to position by the passed origin and that is what you missed

Reply:
that is not a feature. we don't do that.
originX and originY represent the anchor point of the object that is located at top/left. They don't represent anything else. They can't also be used to say that since a bounding box is smaller the bbox left/top corner will be aligned wit the top/left corner of the elements, or in another case bottom/right. That is not originX/originY.

Reply:
I wrote group layout to fix v5 behavior not to organize it
Centering it makes no sense and again this is why this PR should be blocked until you remove originX/Y
If you do not agree with me go ahead
I don't approve this direction.

Reply:
Well you didn't comunicate that clearly when you organize the group rewrite, at all.
Please try to explain what issue the centering causes apart solving the issue of shifting objects.